scripts/run.bash

```
@@ -39,54 +39,187 @@ join_by() {
-  local debian_dev_states_for_dom0=(
-    "catalog.debian.minimal-templates-installed"
+run_states_for_dev_at_dom0() {
+  local states_for_dom0_list=(
     "tpl-dev-debian.clone"
+    "tpl-dev-fedora.clone"
   )
-  sudo qubesctl --show-output --targets=dom0 state.apply "$(join_by ',' "${debian_dev_states_for_dom0[@]}")"
-  # dev templates - debian - templates
-  local debian_dev_states_for_templates=(
-    # "catalog.debug.print_vars"
+  local states_for_dom0="$(join_by ',' "${states_for_dom0_list[@]}")"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)